### DISCUSSION TITLE
Several "Level2"

### DISCUSSION THREAD

**Discussion by manuelrn** [quietvoid/dovi_tool#142] [General]
Hi!

When I work with an RPU (normally converted to profile 8) and I look at the metadata of a frame, I always notice that in many occasions it has several `Level2` instead of just one as it happens with the rest of the `Level`, and I have always wondered if that is normal.

So I thought I would open a Discussion and ask about it, and see if anyone knows anything about it.

Regards!

**Comment by saindriches** [comment]
Level 2 / 8 / 10 are related to target displays and trims, up to 4 trims are allowed in RPU.

**Comment by manuelrn** [comment]
Okay, thanks @saindriches!
